example/webpackServeBundle.js

```diff
@@ -17,11 +17,9 @@ module.exports = (app) => {
       modules: false
     }
   });
+  const hotMiddleware = webpackHotMiddleware(compiler);
 
-  app.use(middleware);
-  app.use(webpackHotMiddleware(compiler));
-  app.get('/falcor-postman', (req, res) => {
-    res.write(middleware.fileSystem.readFileSync(path.join(__dirname, '/../dist/falcor-postman.html')));
-    res.end();
-  });
+  options.app.use(devMiddleware);
+  options.app.use(hotMiddleware);
+  options.app.use(falcorPostman(options, devMiddleware.fileSystem));
```

Comment:
now we are using the **real** middleware also in dev; what do you think @nickbalestra ?